content-src/experiments/lockbox.yaml

```diff
@@ -0,0 +1,162 @@
+id: 16
+slug: firefox-lockbox
+title: 'Firefox Lockbox'
+# subtitle: 'Take your passwords everywhere'
+order: 3
+is_featured: true
+platforms: ['mobile']
+web_url: https://example.com/web-experiment
+
+description: >
+  Take your passwords everywhere with Firefox Lockbox, an iOS mobile app.
+# warning: >
+#   Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum cum ad deserunt
+#   iusto possimus. Fugiat odit corrupti cumque.
+introduction: >
+  <p>Firefox Lockbox lets you easily access your passwords everywhere. Logins are stored and synced securely, using 256-bit encryption, keeping your information safe. Easily copy username and password details to get into apps and websites. Never again find yourself locked out of your account while on your phone!</p>
+  <p><strong>Firefox Lockbox requires a Firefox Account in order to access your previously saved logins. If you do not have an account set up, get started <a href="https://lockbox.firefox.com/faq.html#how-do-i-enable-sync-on-firefox">here</a>.</strong></p>
+
+image_twitter: /static/images/experiments/lockbox/social/twitter.jpg
+image_facebook: /static/images/experiments/lockbox/social/facebook.jpg
+
+thumbnail: /static/images/experiments/lockbox/icon/thumbnail.png
+gradient_start: '#00BCD8'
+gradient_stop: '#0086DC'
+
+xpi_url: 'https://testpilot.firefox.com/files/experiment-name/latest'
```

Comment:
Should this be /lockbox/ or /@lockbox/ or something instead of "experiment-name"?

Reply:
Although, if you're an iOS app without an add-on, you may be able to drop these values.
You may want to check what Send did in https://github.com/mozilla/testpilot/blob/master/content-src/experiments/send.yaml